protocol/flatbuffers/rpc.fbs

```diff
@@ -27,7 +27,10 @@ union RpcMessage {
 	OpenSerialRequest,
 	CloseSerialRequest,
 	SetWifiRequest,
-	SerialUpdateResponse
+	SerialUpdateResponse,
+	AutoBoneProcessRequest,
+	AutoBoneProcessStatus,
+	AutoBoneEpoch
```

Comment:
Same thing here, this file is for Requests and Responses, as this is what an RPC pattern does: 1 request produces 1 response. There needs to be something that requests this value. If thats not the intended design, lets talk on discord because its likely that we will need to slightly redesign something either in the protocol or in your use of the protocol

Reply:
Resolved via discord conversation: The current protocol has diverged from the traditional Req/Resp pattern and therefore this is not a blocker for merging. We will take another look at the request/response pattern later.